### PR TITLE
[Easy] replace lost network deployment info

### PR DESCRIPTION
**Pull request by bh2smith**
Simply putting back some network info that was lost in #33

### CODE REVIEW

File: networks.json [hunk 1]
```diff
@@ -4,6 +4,78 @@
...
+      "links": {},
+      "address": "0xc8e7354a5455b99a0c42c89ee79b4cca1804e957",
+      "transactionHash": "0x6d57ee69ad1b36e9d4e4e5032efade4002dc87e3d60dfeb0bfe97abe452df17f"
+    },
+    "42": {
+      "events": {},
+      "links": {},
+      "address": "0x8bdd53a53c53fbc40d38231d6219f8cb7b59a704",
+      "transactionHash": "0x6c4ae0abb0355a44b88d90e13a3359cc7d28215afdd65928a898bca1d228d337"
+    }
```
Comment (fleupold) [inline networks.json:19]: These are the Migration.sol addresses not `BridgedTokenOWL`

**Review by fleupold — CHANGES_REQUESTED** [review]

File: networks.json (hunk 1 above)
Comment (bh2smith) [reply networks.json:19]: I coppied this directly from the lost info in #33...

Comment (bh2smith) [reply networks.json:19]: I see, there is another line at the top I missed.

Comment (bh2smith) [reply networks.json:27]: Fixed.

**Review by fleupold — APPROVED** [review]

**Comment by fleupold** [comment]
Thanks for fixing this!

**Pull request merged**